src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,33 @@
+package mate.academy.service
+
+import kotlin.random.Random
+import mate.academy.model.Color
+import mate.academy.model.Circle
+import mate.academy.model.Figure
+import mate.academy.model.IsoscelesTrapezoid
+import mate.academy.model.Rectangle
+import mate.academy.model.RightTriangle
+import mate.academy.model.Square
+
+const val FROM = 1.0
+const val TO = 10.0
+const val DEFAULT_RADIUS = 10.0
+const val FIGURE_COUNT = 5
+
+class FigureSupplier {
+    fun getRandomFigure(): Figure {
+        val colorSupplier = ColorSupplier()
+        return when (Random.nextInt(FIGURE_COUNT)) {
+            0 -> Circle(Random.nextDouble(FROM, TO), colorSupplier.getRandomColor())
+            1 -> RightTriangle(Random.nextDouble(FROM, TO), Random.nextDouble(FROM, TO), colorSupplier.getRandomColor())
+            2 -> Rectangle(Random.nextDouble(FROM, TO), Random.nextDouble(FROM, TO), colorSupplier.getRandomColor())
+            //3 -> Square(Random.nextDouble(FROM, TO), colorSupplier.getRandomColor())
```

Comment:
what is the reason this case is commented?

Reply:
checkstyle fails, it shows an error, that number "3" is a "magic number" here and should be replaced with const val, but previous numbers 0, 1 and 2 don't confuse it. I think, that it is invalid settings of rules of the check. Because it is a correct usage of "when" statement syntax. I tried to add more conditions (4, 5) just to see how it behaves, and it recognizes all of them as magic numbers. So, it allows only three number cases in "when" statement.

Reply:
Then, it probably makes sense to map a randomized number to the figure type or use any other trick. Commenting this line brokes the logic and currently Square cannot be returned from the method